src/Microsoft.VisualStudio.Editors/PropPages/AccessibleComboBox.vb

```diff
@@ -0,0 +1,15 @@
+﻿' Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+Imports System.Windows.Forms
+
+Namespace PropPages
+
+    Public Class AccessibleComboBox
+        Inherits ComboBox
+
+        Protected Overrides Sub OnSelectedIndexChanged(e As EventArgs)
+            MyBase.OnSelectedIndexChanged(e)
+            AccessibilityObject.GetType().GetMethod("RaiseAutomationNotification")?.Invoke(AccessibilityObject, new object() {2,4,  Items.Item(SelectedIndex)})
```

Comment:
@MiYanni this is VB so points 2&4 don't work here :(

Reply:
added comments

Reply:
OHHHH. Right, VB. So, the casing is what threw my brain off. In VB, it would normally be like:

`New Object() { 2, 4, Items.Item(SelectedIndex) }`

It is a convention thing for the language [as seen here](http://makble.com/how-to-create-object-array-in-vbnet).
BUT! [According to this](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/collection-initializers/), you can just use:

`{ 2, 4, Items.Item(SelectedIndex) }` 

without `New Object()`. You'd have to check that it compiles.